api/tests/Serializer/Normalizer/RelatedCollectionLinkNormalizerTest.php

```diff
@@ -48,7 +48,7 @@ class RelatedCollectionLinkNormalizerTest extends TestCase {
     private MockObject|PropertyAccessorInterface $propertyAccessor;
     private EntityManagerInterface|MockObject $entityManager;
 
-    private ?FilterInterface $filterInstance;
+    private null|DateFilter|SearchFilterInterface $filterInstance;
```

Comment:
This test failed locally because the filters are not under a common interface anymore...?